### PR TITLE
hot fix for non string

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

**Review by wbreza — APPROVED** [review]

File: cli/azd/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Release History
 
+## 0.3.0-beta.5 (2022-10-21)
+
+### Other Changes
```
[the remark this answers is not in the document]
Comment (vhvb1989) [reply cli/azd/CHANGELOG.md:5]: no, thank you. Will remove

**Comment by vhvb1989** [comment]
> Just to confirm, are we not using NewAzureCLICredential in cli_test.go like we did in PR 979?

Not really part of the fix.  That's more like a test update..  So, I am leaving that one out of this unique hotfix

File: cli/azd/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 0.3.0-beta.5 (2022-10-21)
+
+### Bugs Fixed
+
+- [[#979]](https://github.com/Azure/azure-dev/pull/979) Fix provisioning template with non string outputs.
```
Comment (weshaggard) [inline cli/azd/CHANGELOG.md:7]: Is this correct or was it a copy/paste from the beta.4 build?

Comment (vhvb1989) [reply cli/azd/CHANGELOG.md:7]: both :)  . It is correct and copy-pasted.

The beta.4 was created to fix this issue, but ended up bringing a new bug.

beta.5 is beta.4 minus the bug introduced on beta.4 (deprecated).  beta.5 was created from a buxfix/ branch, having only the cherry pick for the fix

**Pull request merged**